mirrord/layer/src/error.rs

```diff
@@ -14,8 +14,12 @@ use tracing::{error, info};
 use super::HookMessage;
 use crate::tcp_steal::http_forwarding::HttpForwarderError;
 
+/// Error codes from [`libc`] that are **not** hard errors, meaning the operation may progress.
+///
+/// Prefer using the [`should_ignore`] instead of relying on this constant.
```

Comment:
```suggestion
/// Prefer using [`should_ignore`] instead of relying on this constant.
```

Reply:
Also can't this be enforced with public/private?

Reply:
Changed this to be inside of a private module, now you can only use the [`should_ignore`] (also changed the name to `is_ignored_code`, as we have a `should_ignore` macro for files).

What do you think?

Reply:
Neat